microovn/node/node.go

```diff
@@ -40,6 +40,13 @@ func DisableService(s *state.State, service string) error {
 
 	if SrvName(service) == SrvCentral {
 		err = snap.Stop("ovn-ovsdb-server-nb", true)
+		centrals, err := FindService(s,service)
+		if err!=nil {
+			return err
+		}
+		if len(centrals)==1 {
+			return errors.New("You cannot delete the final enabled central service")
+		}
 		if err != nil {
 			return err
 		}
```

Comment:
Isn't this err handling superfluous? There doesn't seem to be anything else that could produce error after the `FindService` call and that's already handled.